app/src/main/java/ir/kindnesswall/view/main/more/MoreFragment.kt

```diff
@@ -4,104 +4,195 @@ import android.os.Bundle
-                    })
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        view as ComposeView
+        view.setViewCompositionStrategy(ViewCompositionStrategy.DisposeOnViewTreeLifecycleDestroyed)
+        view.setContent {
+            MdcTheme {
+                Scaffold(
+                    topBar = {
+                        TopAppBar(title = { Text(text = stringResource(R.string.more_items)) })
+                    }
+                ) {
+                    Column(
+                        modifier = Modifier
+                            .fillMaxSize()
+                            .verticalScroll(rememberScrollState())
+                    ) {
+
+                        val isLoggedIn = UserInfoPref.asLiveData(UserInfoPref::bearerToken)
+                            .observeAsState().value.isNullOrBlank().not()
+
+                        if (isLoggedIn)
+                            Header(
+                                name = UserInfoPref.asLiveData(UserInfoPref::name).observeAsState().value ?: "",
+                                avatarUrl = UserInfoPref.asLiveData(UserInfoPref::image).observeAsState().value ?: "",
+                                phoneNumber = UserInfoPref.asLiveData(UserInfoPref::phoneNumber).observeAsState().value
+                                    ?: ""
+                            )
+
+                        if (isLoggedIn)
+                            Item(
+                                textResId = R.string.my_profile,
+                                imgResId = R.drawable.ic_profile_placeholder_gary
+                            ) {
+                                UserProfileActivity.start(
+                                    requireContext(),
+                                    UserInfoPref.getUser()
+                                )
+                            }
+
+                        if (isLoggedIn)
+                            Item(
+                                textResId = R.string.show_number,
+                                imgResId = R.drawable.ic_baseline_phone_iphone_24
+                            ) { openPhoneVisibilityDialog() }
+
+                        if (UserInfoPref.asLiveData(UserInfoPref::isAdmin).observeAsState().value == true)
+                            Item(
+                                textResId = R.string.check_submitted_gifts,
+                                imgResId = R.drawable.ic_check_and_review
+                            ) {
+                                ReviewGiftsActivity.start(requireContext())
+                            }
+
+                        /*Item(
+                            textResId = R.string.bookmarks,
+                            imgResId = R.drawable.ic_bookmark_gray
+                        ) {}*/
+
+                        /*Item(
+                            textResId = R.string.setting,
+                            imgResId = R.drawable.ic_settings_gray
```

Comment:
Is there a special reason to keep this commented codes? Can we remove them?

Reply:
The commented items came from its legacy `.xml`. They were gone in a commit but I didn't why.
[1](https://github.com/kindnesswall/KindnessWall.KotlinAndroidApp/blob/a6e2878755c9ab8269b8f7e07ec0fe077c479b83/app/src/main/res/layout/fragment_more.xml#L207) or [2](https://github.com/kindnesswall/KindnessWall.KotlinAndroidApp/blob/a6e2878755c9ab8269b8f7e07ec0fe077c479b83/app/src/main/res/layout/fragment_more.xml#L231) or [3](https://github.com/kindnesswall/KindnessWall.KotlinAndroidApp/blob/a6e2878755c9ab8269b8f7e07ec0fe077c479b83/app/src/main/res/layout/fragment_more.xml#L282)

So I decided to keep them and focus on migrating to Compose.

If you say they're unused we can remove them.